README.md

```diff
@@ -4,18 +4,18 @@ This repo contains PyTorch deep learning models for document classification, imp
 
 ## Models
 
-+ [Kim CNN](./kim_cnn/): Baseline convolutional neural network for sentence classification [(Kim, EMNLP 2014)](http://www.aclweb.org/anthology/D14-1181)
-+ [Conv-RNN](./conv_rnn/): Convolutional RNN [(Wang et al., KDD 2017)](https://dl.acm.org/citation.cfm?id=3098140)
-+ [HAN](./han/): Hierarchical Attention Networks [(Zichao, et al, NAACL 2016)](https://www.cs.cmu.edu/~hovy/papers/16HLT-hierarchical-attention-networks.pdf)
-+ [LSTM-Reg](./lstm_regularization/): Standard LSTM with Regularization [(Merity et al.)](https://arxiv.org/abs/1708.02182)
-+ [XML-CNN](./xml_cnn/): CNNs for Extreme Multi-label Text Classification [(Liu et al., SIGIR 2017)](http://nyc.lti.cs.cmu.edu/yiming/Publications/jliu-sigir17.pdf)
++ [Kim CNN](models/kim_cnn/): CNNs for sentence classification [(Kim, EMNLP 2014)](http://www.aclweb.org/anthology/D14-1181)
```

Comment:
Need to rewrite the README's

Reply:
Yes, I agree. Though this is something I would like to do after all the refactoring is done. Plus, we would have to write a runbook for NAACL too.